release/ansible/Dockerfile.rhel8

```diff
@@ -20,18 +21,20 @@ ENV HOME=/opt/ansible \
     USER_NAME=ansible \
     USER_UID=1001
 
+# get artifacts from cachito
+COPY $REMOTE_SOURCE/cachito-gomod-with-deps/app $REMOTE_SOURCE_DIR
+WORKDIR $REMOTE_SOURCE_DIR/app
```

Comment:
Do we need to add an `ARG` or `ENV` directive that can be overridden for this `$REMOTE_SOURCE` env var? How is it injected now?

Reply:
I don't think we override these ever. As far as I'm aware this is provided by the build system, and always refers to the paths provided by cachito. In STF we have never provided an override interface for these values.